routes/error-tracker.js

```diff
@@ -49,6 +49,47 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @desc converts stack traces and standardizes them to chrome like.
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  let chromeStackTraceRegex = /^\s*at (?:([^]*) )?([^]+):(\d+):(\d+)/m;
+  let match = chromeStackTraceRegex.test(exception);
+  if (match) {
+    exception = exception.substring(exception.indexOf('\n'));
+    let exceptions = exception.split('\n');
+    let validExceptions = exceptions.filter(function(value) {
+      return chromeStackTraceRegex.test(value);
+    });
+    exception = validExceptions.join('\n');
+    return exception;
+  } else {
+    let mozillaSafariStackTraceRegex = /^([^@\n]*)@(.+):(\d+):(\d+)$/gm;
+    let exceptions = [];
+    let otherMatch;
+    while (otherMatch = mozillaSafariStackTraceRegex.exec(exception)) {
+      exceptions.push(otherMatch[0]);
+    }
+    let validExceptions = exceptions.map(safariOrMozillaToChrome);
```

Comment:
This can be moved into the while loop above. While we're there, we can drop the function entirely (we already have the parsed match we can construct the proper frame with).